aiohttp_sse/__init__.py

```diff
@@ -25,7 +29,7 @@ def hello(request):
 
     DEFAULT_PING_INTERVAL = 15
 
-    def __init__(self, *, status=200, reason=None, headers=None):
+    def __init__(self, *, status=200, reason=None, headers=None, request=None):
```

Comment:
Oh, now got your previous message about `__init__`. What do you think about creating function, like

```python
def sse_response(self, request, *, status=200, reason=None, headers=None)
    response = EventSourceResponse(...)
    response.start(request)
    return response
```
kind of like `json_response` from `aiohttp`. This way we will not break interface of EventSourceResponse/Response class and still possible to do context manager:
```python
async with sse_response(request, ...) as response:
    response.send(...)

```